play_yt.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+youtube-dl -o "/tmp/%(id)s.%(ext)s" --extract-audio --audio-format mp3 $1
```

Comment:
I think we should surround all the `$1` usages with "quotes". Sanitising the id as kathryn suggesting is a nice safeguard, but using `"$1"` means that bash will handle all the of sanitizing and stuff for us!

e.g. 
```
 $ foo='here is a "spooky" ../ string!'
 $ echo "$foo"
here is a "spooky" ../ string!
 $ cat "$foo"
cat: 'here is a "spooky" ../ string!': No such file or directory
```

Reply:
i didn't suggest any fix just pointed out the problem.